src/src/Form/AdType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\Ad;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class AdType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('title')
+            ->add('description')
+            ->add('price')
```

Comment:
Missing $thumbnails_urls field  !

Reply:
Come in next PR as I said ;)

<img width="390" alt="image" src="https://user-images.githubusercontent.com/40914400/197736180-64f29460-b712-4843-b2d5-c4cab17f40ce.png">

Reply:
Ok